server/Html.js

```diff
@@ -14,6 +14,10 @@ export default class Html extends React.Component {
     return (
       <html lang="fi">
         <head>
+          <script
```

Comment:
Why not use raven's NPM package? If need be it can be included outside regular bundle from static assets with some webpack trickery.

Reply:
Done